src/transform/CollapseTable.css

```diff
@@ -0,0 +1,140 @@
+/* Examples:
+  - Barack Obama's collapsed infobox:
+
+    <div class=app_table_container>
+      <div class='app_table_collapsed_container app_table_collapsed_open'
+        style='display: block'>
+        <strong>Quick facts</strong><span class=app_span_collapse_text>: 44th
+          President of the United States, Vice President …</span>
+      </div>
+      <table class='infobox vcard'
+        style='width: 22em; margin-top: 0px; margin-bottom: 0px; display: none'>
+        <tbody>
+          <tr>
+            <th colspan=2
+              style='text-align: center; font-size: 125%; font-weight: bold; font-size: 130%'>
+              <span class=fn>Barack Obama</span>
+            </th>
+          </tr>
+          ...
+        </tbody>
+      </table>
+      <div class='app_table_collapsed_bottom app_table_collapse_icon'
+        style='display: none'>
+        Close
+      </div>
+    </div>
+
+  - Frankenstein's monster's expanded infobox:
+
+    <div class=app_table_container>
+      <div class='app_table_collapsed_container app_table_collapse_close app_table_collapse_icon'
+        style='display: block'>
+        <strong>Quick facts</strong><span class=app_span_collapse_text>: Created
+          by, Information …</span>
+      </div>
+      <table class=infobox
+        style='width: 22em; border-spacing: 2px 5px; margin-top: 0px; margin-bottom: 0px; display: block'>
+        <tbody>
+          <tr>
+            <th colspan=2
+              style='text-align: center; font-size: 125%; font-weight: bold; background: #dedee2;'>
+              Frankenstein's monster
+            </th>
+          </tr>
+          ...
+        </tbody>
+      </table>
+      <div class='app_table_collapsed_bottom app_table_collapse_icon'
+        style='display: block'>
+        Close
+      </div>
+    </div>
+
+  - Dracula's collapsed notes and references:
+
+    <div class=app_table_container>
+      <div class='app_table_collapsed_container app_table_collapsed_open'
+        style='display: block'>
+        <strong class=app_table_collapsed_caption>Tap to expand</strong>
+      </div>
+      <div class='reflist references-column-width'
+        style='column-width: 30em; list-style-type: decimal; padding: 4px; display: none'>
+          ...
+      </div>
+      <div class='app_table_collapsed_bottom app_table_collapse_icon'
+        style='display: none'>
+        Close
+      </div>
+    </div>
+*/
+
+.content table.infobox {
+  font-size: 100%;
+  border-style: none !important;
+}
+
+table.infobox caption {
+  text-align: center;
+  font-weight: bold;
+}
+
```

Comment:
Do we need the `figcaption` class I see [here](https://gerrit.wikimedia.org/r/#/c/349373/1/app/src/main/assets/applib.css)?